### DISCUSSION TITLE
feat: Seek before chapter beginning to end of previous chapter

### DISCUSSION THREAD

**Discussion by somombo** [PaulWoitaschek/Voice#1758] [Ideas]
In v6 of the app I could keep pressing the "seek backwards" button until it goes to the end of the previous chapter. This was a very practical and useful feature please consider bringing it back, otherwise I am forced to use v6 and forgo other bug fixes and feature updates

**Comment by PaulWoitaschek** [comment]
This sounds like a bug. Can you please file it as a bug using the template?

**Reply by PaulWoitaschek** [reply]
Btw, you don't need to leave bad reviews on google play, I'll also help without them 😊

**Reply by somombo** [reply]
oh sorry. Your breed of dedication to actually considering feedback is not common on the playstore. So I just assumed... I cannot find my initial bad review.. was looking for it so I could delete in anycase, I have now added a plublic 5star review for you. Thanks for getting on top of this so quickly. I really appreciate it! Keep up the great work!

**Reply by somombo** [reply]
> This sounds like a bug. Can you please file it as a bug using the template?

Should I still open a separate issue? @PaulWoitaschek

**Reply by PaulWoitaschek** [reply]
Yes, that would be great!

You can either do it from the app (then some stuff is prefilled) or directly through GitHub 
https://github.com/PaulWoitaschek/Voice/issues/new?assignees=&labels=bug&template=bug.yml
![Screenshot_20230403-174113](https://user-images.githubusercontent.com/743462/229559939-2a231b4c-d0c2-422b-8ca5-62c79280264e.png)

**Reply by somombo** [reply]
I've posted issue #1800 @PaulWoitaschek
